test/relayer_chain_test.go

```diff
@@ -115,7 +115,11 @@ func chainTest(t *testing.T, tcs []testChain) {
 	ctx, cancel := context.WithCancel(context.Background())
 	defer cancel()
 
-	_ = relayer.StartRelayer(ctx, src, dst, 2*cmd.MB, 5)
+	filter := &relayer.ChannelFilter{
+		Rule:        "",
```

Comment:
This and the two other instances in this file are explicitly setting the zero values, which is not necessary.